encord/objects/ontology_structure.py

```diff
@@ -140,7 +149,9 @@ def to_dict(self) -> Dict[str, List[Dict[str, Any]]]:
         ret["classifications"] = ontology_classifications
         for ontology_classification in self.classifications:
             ontology_classifications.append(ontology_classification.to_dict())
-
+        skeleton_templates: List[Dict[str, Any]] = list()
+        skeleton_templates = [{"template": template.to_dict()} for template in self.skeleton_templates.values()]
```

Comment:
I wonder why are these two separate lines? Or, perhaps, even all three of them?

Reply:
This follows convention from the above line where "ontology_classifications: List[Dict[str, Any]] = list()"